plugins/inputs/snmp/snmp_mocks_generate.go

```diff
@@ -0,0 +1,97 @@
+// +build generate
```

Comment:
I built this generator to make management of the mocked commands easier. Having to manually add the command & output to the test file was becoming a PITA.